src/inlineExpression.tsx

```diff
@@ -84,7 +83,7 @@ export class RenderedExpression extends Widget {
 
     // Create renderer
     const renderer = this.rendermime.createRenderer(mimeType);
-    layout.widget = renderer;
+    if (layout) layout.widget = renderer;
     console.assert(renderer.isAttached, 'renderer was not attached!', renderer);
```

Comment:
Many of these calls consistently fail, and aren't actually a problem. I think that we should not show console errors unless there is a problem. We could also put them behind a dev flag if we want.